.snippets/code/builders/build/eth-api/dev-env/brownie/terminal/new-account.md

```diff
@@ -0,0 +1,11 @@
+<div id="termynal" data-termynal>
+    <span data-ty="input">brownie accounts new alice</span>
+    <span data-ty>Brownie v1.19.1 - Python development framework for Ethereum</span>
+    <span data-ty="progress"></span>
```

Comment:
for breaks you can just use `<br>` there shouldn't be any termynal snippets with `progress` used right now, that's typically used to display a progress bar